internal/command/project.go

```diff
@@ -345,7 +346,7 @@ func (c *Commands) RemoveProject(ctx context.Context, projectID, resourceOwner s
 	}
 
 	for _, grantID := range cascadingUserGrantIDs {
-		event, _, err := c.removeUserGrant(ctx, grantID, "", true)
+		event, _, err := c.removeUserGrant(ctx, grantID, gu.Ptr(""), true, false, nil)
```

Comment:
You should have the resourceOwner of the grant, if it is queried.

Reply:
Maybe that would improve the eventstore queries but that's currently not the behavior on main AFAICT.
BTW, otherwise, we could just pass nil instead of gu.Ptr(""), I think :smile: 
Fine by me if we want to change that, but as this is a deprecated method, I'm in favor of not touching it too much in the scope of this PR.
WDYT?

Reply:
discussed